### DISCUSSION TITLE
월드 크리에이터에게 포털 기능 제공은 계획이 있으신지 문의드립니다.

### DISCUSSION THREAD

**Discussion by kimkanu7** [naverz/zepeto-studio-kor#205] [4. General]
안녕하세요^^
질문은 아래와 같습니다.

월드 크리에이터에게 포털 기능 제공은 계획이 있으신지 문의드립니다.

**Comment by beanyh** [comment]
https://github.com/naverz/zepeto-studio-kor/discussions/203 질문과 동일한 것 같습니다~ 
아직 제공 계획을 가지고 있지 않은 점 양해 부탁드립니다.
